projects/robots/bluebotics/shrimp/worlds/shrimp.wbt

```diff
@@ -352,5 +354,5 @@ DEF SPHERES Solid {
   boundingObject USE BALLS
 }
 Shrimp {
-  translation 0.02 0 4.5
+  translation -4.5 0 0.02
```

Comment:
```suggestion
  translation -4.5 -0.02 0
```

Reply:
This might be wrong, on master the robot is on the ground whereas in this branch it floats slightly.